crates/xlayer/db/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "xlayer-db"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[dependencies]
+reth-db.workspace = true
```

```suggestion
reth-db = { workspace = true, features = ["mdbx"] }
```

Otherwise `cargo c -p xlayer-db` on target crate will fail.

---

ok